terragraph_planner/data/template.yaml

```diff
@@ -112,189 +110,150 @@ LINE_OF_SIGHT:
-  # LOS. Lower values can provide more candidate links at greater risk that
-  # those links do not have actual LOS during field surveys. 0 for direct LOS,
-  # 1 for LOS with Fresnel zone clearance. Minimum level of confidence MUST be
-  # in [0, 1].
-
+  # [Float] Minimum level of confidence required to declare a valid LOS. Lower
+  # values can provide more candidate links at greater risk that those links do
+  # not have actual LOS during field surveys. 0 for direct LOS, 1 for LOS with
+  # complete Fresnel zone clearance. Minimum level of confidence must be in
+  # [0, 1].
   LOS_CONFIDENCE_THRESHOLD: 0.8
 
   SITE_DETECTION:
     # [Bool] Enable candidate DN site generation on each building. The DN will
     # be deployed at the same location as the CN with the greatest potential
-    # line of sight.
-
+    # line-of-sight.
     DN_DEPLOYMENT: True
 
     # [Bool] Enable candidate site generation at the highest/centers/corners of
     # each building.
-
     DETECT_HIGHEST: True
     DETECT_CENTERS: False
     DETECT_CORNERS: True
 
-    # [Optional][Float] The maximum angle, in degrees, that a building corner can
-    # be to qualify for site deployment.
+    # [Optional][Float] The maximum angle, in degrees, that a building corner
+    # can be to qualify for site deployment.
     MAX_CORNER_ANGLE: null
 
-  # [Optional] Number of the processors used for LOS computation. If null is given,
-  # the planner will use as many as possible.
+  # [Optional] Number of the processors used for LOS computation. If null, the
+  # planner will use as many as possible.
   NUM_PROCESSORS: null
 
 RADIO:
-  # [Float] Frequency (in MHz) at which these sectors operate.
-
+  # [Float] Frequency (in MHz) at which the sectors operate.
   CARRIER_FREQUENCY: 60000.0
 
   # [Float] Noise figure (dB)
-
   NOISE_FIGURE: 7.0
 
   # [Float] Thermal noise power (dBm)
-
   THERMAL_NOISE_POWER: -81.0
 
-  # [Optional][Float] Maximum Equivalent/Effective Isotropically Radiated
-  # Power in dBm.
-
+  # [Optional][Float] Maximum Equivalent/Effective Isotropically Radiated Power
+  # in dBm.
   MAXIMUM_EIRP: null
 
   # [Float] Rain rate in mm/hr for the planning region.
-
   RAIN_RATE: 30.0
 
   # [Float] The percentage of time in one year that each line-of-sight (LOS)
   # link will be live, such as 95, 99, 99.99, etc.
   LINK_AVAILABILITY_PERCENTAGE: 99.9
 
-  # A list of device data that could be used in the plan.
-  # Add more devices after the '-' symbol.
-
+  # A list of device data that could be used in the plan. Add each device after
+  # the '-' symbol. This list is empty by default. For each entry, DEVICE_SKU
+  # must be provided, the remaining parameters have defaults as shown (except
+  # for file paths which have null defaults).
   DEVICE_LIST:
-    # [String] The device SKU or Name that used to identify
-    # the hardware.
-
+    # [Required][String] The device SKU or Name that used to identify the
```

Comment:
Remove "that" unless you actually mean "that used to" as in "this is how it worked at some point in the past". Actually, I think saying "... or Name that identifies the..." is even better (assuming I read your intent correctly).